pkg/service/service.go

```diff
@@ -45,8 +46,20 @@ type ServicePlanParameter struct {
 	Description     string
 	HasDefaultValue bool
 	Default         string
-	Type            string
-	Required        bool
+	odoutil.Validatable
+}
+
+// NewServicePlanParameter creates a new ServicePlanParameter instance with the specified state
+func NewServicePlanParameter(name, typeName, defaultValue string, required bool) ServicePlanParameter {
```

Comment:
Actually I don't think this should be here, since it's only used for tests - so why not add it to one of the test util files?

Reply:
This point is debatable. This function could be used outside of tests, even though it's not currently, especially since with these changes, you need to properly initiate the `Validatable` part of the `ServicePlanParameter`.

Reply:
I doesn't feel right to me, but I see your point